frontend/ui/elements/page_turns.lua

```diff
@@ -122,6 +122,21 @@ local PageTurns = {
                     end,
                 })
             end,
+        },
+        {
+            text = _("Also invert document-related dialogs"),
+            checked_func = function()
+                return G_reader_settings:isTrue("invert_ui_layout_mirroring")
+            end,
+            enabled_func = function()
```

Comment:
I would propose
```lua
text = _("Invert document-related dialogs"),
checked_func = function()
    return G_reader_settings:isTrue("invert_ui_layout_mirroring") and
        ReaderUI.instance.view.inverse_reading_order
end,
```
So, it will be disabled and unchecked without `Invert page turn taps and swipes` and "Also" word wouldn't be needed.

Reply:
Good point!

Reply:
However then it won't actually show the state of this setting leading to confusion. (I think it is disabled, then enable inverse_reading_order and this becomes enabled without me realizing)

Reply:
Yes, it doesn't show the state of the setting, but it shows (unchecked) that currently the dialogs are not inverted.